jobs/azure_cpi/templates/cpi.json.erb

```diff
@@ -75,6 +75,10 @@
     params['cloud']['properties']['azure']['azure_stack']['endpoint_prefix'] = p('azure.azure_stack.endpoint_prefix')
   end
 
+  if_p('azure.isv_tracking_guid') do |isv_tracking_guid|
+    params['cloud']['properties']['azure']['isv_tracking_guid'] = isv_tracking_guid
```

Comment:
How about checking whether it's really a GUID? At least the length should be 36?

Reply:
thanks. will change.